Dockerfile

```diff
@@ -66,24 +66,27 @@ RUN chmod 644 $PROJECTOR_DIR/ide/projector-server/lib/*
 
 FROM eclipse-temurin:${containerJdkVersion}-jdk-jammy
 
-# Add custom CA certs
+# Add custom CA certificates
 ARG extraCaCertsDir
 ADD ${extraCaCertsDir} /usr/local/share/ca-certificates/
 
 RUN true \
 # Any command which returns non-zero exit code will cause this shell script to exit immediately:
-   && set -e \
+    && set -e \
 # Activate debugging to show execution details: all commands will be printed before execution
-   && set -x \
+    && set -x \
+    && update-ca-certificates \
```

Comment:
You're already running this on line 87. I think this should be run after you install ca-certificates anyway?

Reply:
Done